### ISSUE TITLE
How to add callback upon chart interacton

### ISSUE THREAD

**Issue by minggo** [Jpadilla1/react-native-ios-charts#25]
As mentioned in [this documentation](https://github.com/PhilJay/MPAndroidChart/wiki/Interaction-with-the-Chart), chart can add callback listener on chart interaction. But i don't find any information about it in `react-native-ios-charts`.

**Comment by minggo** [comment]
@Jpadilla1 any idea?

**Comment by Jpadilla1** [comment]
Yeah the callback listeners are missing. I'll see when I can implement them.

**Comment by minggo** [comment]
@Jpadilla1 Thanks.

**Comment by paulmichalak** [comment]
@Jpadilla1 Any chance there is progress on this?

**Comment by SafiMohammed** [comment]
Is there any updates regarding the callback in chart interaction?

**Comment by prithsharma** [comment]
Any updates on this?
